services/fuzzmanager/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM python:3.6-slim-stretch
+
+LABEL maintainer Gary Kwong <gkwong@mozilla.com>
+
+RUN useradd -d /home/fuzzmanager -s /bin/bash -m fuzzmanager
+WORKDIR /home/fuzzmanager
+ENV DEBIAN_FRONTEND noninteractive
+
+COPY recipes /tmp/recipes
+
+RUN \
+  echo "deb http://ftp.us.debian.org/debian testing main contrib non-free" >> /etc/apt/sources.list \
+  && apt-get update -qq \
+  && apt-get install -y -qq --no-install-recommends --no-install-suggests \
+    apache2-utils \
+    apt-utils \
+    git \
+    locales \
+  && echo "en_US.UTF-8 UTF-8" > /etc/locale.gen \
+  && locale-gen en_US.UTF-8 \
+  && cd /tmp/ \
+  && LANG=en_US.UTF8 LC_ALL=en_US.UTF-8 \
+     /tmp/recipes/fuzzmanager.sh \
```

Comment:
I overlooked that on the first review. If you execute it here it will run under the root user with "/root" as home folder and FM gets installed there.